src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -516,6 +551,113 @@ class CalculatorForm extends React.Component {
     );
   }
 
+  renderExtensionBeneficiaryZIP() {
+    const { profile, inputs, onShowModal } = this.props;
+    const extensions = this.getExtensions();
+
+    let amountInput;
+    let extensionSelector;
+    let extensionOptions;
+    let zipcodeRadioOptions;
+
+    if (extensions.length > 0) {
+      extensionOptions = [{ value: '', label: 'Please choose a location' }];
+      extensions.forEach(extension => {
+        extensionOptions.push({
+          value: extension.zip,
+          label: extension.institution,
+        });
+      });
+      extensionOptions.push({ value: 'other', label: 'Other...' });
+
+      zipcodeRadioOptions = [
+        {
+          value: 'yes',
+          label: profile.attributes.name,
+        },
```

Comment:
`zipcodeRadioOptions` always includes this first option, so could initialize above:

```
const zipcodeRadioOptions = [{
  value: 'yes',
  label: profile.attributes.name,
}];
```

And then push the appropriate option in either condition block:

```
zipcodeRadioOptions.push({ value: 'extension', label: 'An extension campus' });
```